### ISSUE TITLE
Questions about choosing loss scale parameters

### ISSUE THREAD

**Issue by minooisbusy** [cvg/pixloc#54]
Hello, and thank you for the great works.

I have some questions about the loss function.

As written in the paper, the loss for the training network uses Huber loss (pseudo-Hubber loss to be precise), and the robust function of iterative solutions uses alpha = 0.

I have a questions here about setting the scale parameters. 
1. The scale parameter c is 0.1 in the previous loss and 2 in the later loss. Can you explain why you chose each scale parameter?
2. In the function `reprojection_error(T_r2q)`, variable `err` is passed to `scaled_barron()`, dividing the returned value by 4. Is there a special reason for this value to be divided by 4?
